.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+# This file was generated by the excavator check 'excavator/manage-circleci' as specified in .circleci/template.sh.
+# To request a modification to the general template, file an issue on Excavator.
+# To manually manage the CircleCI configuration for this project, remove the .circleci/template.sh file.
+
+version: 2
+jobs:
+  build:
+    docker:
+      - image: ellisjoe/openjdk-java-8-openssl:0.0.2
+    resource_class: xlarge
+    environment:
+      GRADLE_OPTS: -Dorg.gradle.console=plain -Dorg.gradle.internal.launcher.welcomeMessageEnabled=false
```

Comment:
If you use the mega `com.palantir.baseline`, you get `com.palantir.baseline-circleci` which will detect these:

    CIRCLE_TEST_REPORTS: /home/circleci/junit
    CIRCLE_ARTIFACTS: /home/circleci/artifacts